test/utils.js

```diff
@@ -165,3 +174,18 @@ module.exports = {
   braveYoutubeOwner,
   braveYoutubePublisher
 }
+
+function createSurveyor () {
```

Comment:
I would prefer that we make probi and votes arguments

Reply:
that makes sense.